src/pgduckdb_metadata_cache.cpp

```diff
@@ -109,8 +109,9 @@ BuildDuckdbOnlyFunctions() {
 	 * each of the found functions is actually part of our extension before
 	 * caching its OID as a DuckDB-only function.
 	 */
-	const char *function_names[] = {"read_parquet",      "read_csv",   "iceberg_scan", "iceberg_metadata",
-	                                "iceberg_snapshots", "delta_scan", "read_json",    "approx_count_distinct"};
+	const char *function_names[] = {"read_parquet",      "read_csv",     "iceberg_scan",       "iceberg_metadata",
+	                                "iceberg_snapshots", "delta_scan",   "read_json",          "approx_count_distinct",
+	                                "json_exists",       "json_extract", "json_extract_string"};
```

Comment:
It seems this list is missing a bunch of the functions that you added.

Reply:
Added